pantheon-bundle/src/main/java/com/redhat/pantheon/servlet/SimpleFilter.java

```diff
@@ -49,22 +54,30 @@
  *     @Property(name="service.ranking", intValue=1)
  * })
  */
-//@SlingFilter(order=1, description="A Simple Filter")
-//@Property(name="service.vendor", value="The Apache Software Foundation")
+@Component
+@SlingServletFilter(scope = {SlingServletFilterScope.REQUEST},
+                    methods = {"GET","HEAD"})
 public class SimpleFilter implements Filter {
```

Comment:
Could we call this something like `CorsEnablingFilter`? (or a better name)

Reply:
done